src/vaadin-email-field.html

```diff
@@ -8,55 +8,97 @@
 <link rel="import" href="../../polymer/lib/elements/custom-style.html">
 <link rel="import" href="vaadin-text-field.html">
 
-<script>
-  (function() {
-    /**
-     * `<vaadin-email-field>` is a Web Component for email field control in forms.
-     *
-     * ```html
-     * <vaadin-email-field label="Email">
-     * </vaadin-email-field>
-     * ```
-     *
-     * ### Styling
-     *
-     * See vaadin-text-field.html for the styling documentation
-     *
-     * See [ThemableMixin – how to apply styles for shadow parts](https://github.com/vaadin/vaadin-themable-mixin/wiki)
-     *
-     * @memberof Vaadin
-     * @extends Vaadin.TextFieldElement
-     * @demo demo/index.html
-     */
-    class EmailFieldElement extends Vaadin.TextFieldElement {
-      static get is() {
-        return 'vaadin-email-field';
+<dom-module id="vaadin-email-field-template">
+  <template>
+    <style>
+      :host([dir="rtl"]) [part="input-field"] {
+        direction: ltr;
       }
 
-      static get version() {
-        return '2.6.0-alpha4';
+      :host([dir="rtl"]) [part="value"]::placeholder {
+        direction: rtl;
+        text-align: right;
       }
 
-      ready() {
-        super.ready();
-        this.inputElement.type = 'email';
-        this.inputElement.autocapitalize = 'off';
+      :host([dir="rtl"]) [part="input-field"] ::slotted(input)::placeholder {
```

Comment:
Had to split this style with the one above, because this is not valid in Safari and it was not applying for the default input content. There's one issue [opened in the Webkit Bugzilla](https://bugs.webkit.org/show_bug.cgi?id=178237).